doc/src/manual/style-guide.md

```diff
@@ -447,3 +447,43 @@ julia> h(1)
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
 in order to make it easier to use your code.
+
+## Workarounds for named positional arguments
```

Comment:
don't call it a "workaround" because the lack of these is not uncontroversially a deficiency

instead, maybe this section could be subtitled e.g. "Options for referencing positional arguments by name" or something more neutral